options.ixx

```diff
@@ -54,6 +54,7 @@ export struct Options
     int plextor_leadin_retries;
     bool asus_skip_leadout;
     bool disable_cdtext;
+    bool enable_multisession_cdtext;
```

Comment:
Option name is not descriptive enough, it might give users the idea that multisession CD-TEXT is not being read by any drive.
Use force_cdtext_reading and --force-cdtext-reading respectively.